rpm/libngf-qt5.spec

```diff
@@ -12,24 +12,35 @@ BuildRequires:  pkgconfig(Qt5Core)
 BuildRequires:  pkgconfig(Qt5DBus)
 BuildRequires:  pkgconfig(Qt5Qml)
 BuildRequires:  pkgconfig(Qt5Test)
+BuildRequires:  pkgconfig(Qt0Feedback)
```

Comment:
Looks like there's both 0 and 5 available. 5 could be better?

Reply:
Changed.